manifests/f/FSN/WinFeat/1.5.1/FSN.WinFeat.locale.en-GB.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: FSN.WinFeat
+PackageVersion: 1.5.1
+PackageLocale: en-GB
+Publisher: Fireseed Network
```

Comment:
The `Publisher` needs to match the one in the control panel:
![image](https://user-images.githubusercontent.com/52826317/125926486-b187f55d-26d3-496c-a1a7-a8be5e8e762a.png)

```suggestion
Publisher: Fireseed
```